aten/src/ATen/templates/CompositeViewCopyKernels.cpp

```diff
@@ -1,6 +1,7 @@
 #define TORCH_ASSERT_ONLY_METHOD_OPERATORS
 // ${generated_comment}
 
+#include <ATen/InferSize.h>
```

Comment:
Why is this needed?

Reply:
It's for resolving a LazyTensor bug after recent TorchDynamo change. It's from Pytorch team and already approved by ezyang. You can skip this commit.